include/backup_restore.hpp

```diff
@@ -69,7 +77,13 @@ class BackupRestore
                                 const std::string& i_srcPath,
                                 const std::string& i_dstPath);
 
+    // Parsed system JSON config file.
+    nlohmann::json m_sysCfgJsonObj;
+
     // Parsed backup and restore's JSON config file.
-    nlohmann::json m_backupRestoreParsedJson{};
+    nlohmann::json m_backupRestoreCfgJsonObj{};
+
+    // Indicates whether backup and restore is being called already or not.
```

Comment:
> // Indicates whether backup and restore is being called already or not. -> grammar

// Indicates if backup and restore has been performed.

Reply:
Done